django/db/models/functions/datetime.py

```diff
@@ -232,7 +232,7 @@ def as_sqlite(self, compiler, connection, **extra_context):
         return self.as_sql(
             compiler,
             connection,
-            template="STRFTIME('%%Y-%%m-%%d %%H:%%M:%%f', 'NOW')",
+            template="STRFTIME('%%%%Y-%%%%m-%%%%d %%%%H:%%%%M:%%%%f', 'NOW')",
```

Comment:
Appears to be correct as documented… and from your comments on #16111 it sounds like there's already tests in place showing that both %% and %%%% versions are passing?

Reply:
> Appears to be correct as documented… and from your comments on #16111 it sounds like there's already tests in place showing that both %% and %%%% versions are passing?

Yes, it's `test_microseconds` added in 649b28eab6765cd6b2b40c779a18ecafc99b43d7.